### ISSUE TITLE
FileSystemSecurity.xml: The example doesn't compile, contains non-existent method, no typos

### ISSUE THREAD

**Issue by skycommand** [dotnet/dotnet-api-docs#9180]
Hello

I'm reporting a faulty example code in the [FileSystemSecurity Class](https://learn.microsoft.com/en-us/dotnet/api/system.security.accesscontrol.filesystemsecurity?view=net-7.0&source=docs) page for .NET 7.0 (and potentially other versions). The associated source page is [FileSystemSecurity.xml](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Security.AccessControl/FileSystemSecurity.xml)

The code doesn't compile (generating four CS0117 errors) because it calls two non-existent methods: `File.GetAccessControl()` and `File.SetAccessControl()`. 

- In the C# version, the bad calls are on lines 44, 51, 61, and 68.
- In the VB version, the bad calls are on lines 38, 47, 58, and 65.
- In the C++ version, the bad calls are on lines 12, 19, 30, and 37.

Please advise.

**Comment by Jozkee** [comment]
The example needs to be updated to use [FileSystemAclExtensions](https://learn.microsoft.com/dotnet/api/system.io.filesystemaclextensions.getaccesscontrol?view=net-8.0#system-io-filesystemaclextensions-getaccesscontrol(system-io-fileinfo)).
Instead of `File.GetAccessControl(fileName)`, we need `new FileInfo(fileName).GetAccessControl()`.

Although the existing example may still have value for .NET framework users. @gewarren is there a way to make tfm-specific examples?

**Comment by gewarren** [comment]
> @gewarren is there a way to make tfm-specific examples?

@Jozkee Yes you can have have more than one example and just annotate them with the versions they apply to.
